server/src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -372,7 +372,7 @@ public void setOwner(Owner owner) {
     public String toString() {
         String consumerType = (this.getType() != null) ? this.getType().getLabel() : "null";
 
-        return String.format("Consumer [id: %s, uuid: %s, consumerType: %s, name: %s]",
+        return String.format("Consumer [id: %s, uuid: %s, type: %s, name: %s]",
```

Comment:
Why change the name the property? Accident due to mass refactor/rename with IDE?

Reply:
probably. undo-ing